extensions/guacamole-auth-ldap/pom.xml

```diff
@@ -141,11 +141,17 @@
             <scope>provided</scope>
         </dependency>
 
-        <!-- JLDAP -->
+        <!-- Apache Directory LDAP API -->
         <dependency>
-            <groupId>com.novell.ldap</groupId>
-            <artifactId>jldap</artifactId>
-            <version>4.3</version>
+            <groupId>org.apache.directory.api</groupId>
+            <artifactId>api-all</artifactId>
```

Comment:
With things switching from JLDAP to Apache Directory API, the `LICENSE` file and bundled licenses will need to be updated, too:



https://github.com/apache/guacamole-client/blob/8b172a98fc77241664be16e7f40ab4add510c73a/extensions/guacamole-auth-ldap/src/licenses/LICENSE#L232-L243

Reply:
Okay, updated the licenses.